blueprints/ember-cli-changelog/files/config/release.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
   // format: 'YYYY-MM-DD',
   // timezone: 'America/Los_Angeles',
 
-  beforeCommit: function(project, versions) {
+  beforeCommit: function(project, tags) {
     console.log(project);
-    return generateChangelog.call(project);
+    return generateChangelog(project, tags.latest, tags.next);
```

Comment:
The usage in this file was actually buggy, it was intended to be used this way:
```
beforeCommit: generateChangelog
```

We can account for separation of the tags in a wrapper function layer within either project if necessary.